plugins/processors/topk/README.md

```diff
@@ -0,0 +1,25 @@
+# TopK Processor Plugin
+
+The TopK processor plugin is a filter that keeps only the top k values of a given series. It can be tweaked to do its top k computation over a period of time, so spikes can be smoothed out.
+
+### Configuration:
+
+```toml
+[[processors.topk]]
+  period = 10                  # How many seconds between aggregations. Default: 10
+  k = 10                       # How many top metrics to return. Default: 10
+  metric = "mymetric"          # Which metrics to consume. Supports regular expressions. No default. Mandatory
```

Comment:
I needed to check the code on this, but it looks that the normal [measurement filtering](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#measurement-filtering) options work for processors, although perhaps in a somewhat non-intuitive way.

If a metric is filtered out, the processor is bypassed but the metric continues to the outputs[1].  This means that you could use these for selecting the metrics here instead of custom params.  I'm not sure if this is something that should be changed or if the documentation just needs improved.

[1] https://github.com/influxdata/telegraf/blob/master/internal/models/running_processor.go#L39

Reply:
I think the documentation could be improved. Looking around the only documentation regarding processors that I could find was [this](https://docs.influxdata.com/telegraf/v1.1/concepts/aggregator_processor_plugins/). I think I saw the measurements filtering docs before, but I didn't connect them to the processors for some reason, that was my bad.

However, I think that the main docs in docs.influxdata.com should be updated. I just looked around and there isn't a section dedicated to processor configuration, at least not one that is easy to spot. At the very least a link to the measurement filtering should be added IMO.

I'll simply the code so it doesn't use these custom options, since now I now that they are just clutter.

Reply:
Here is the change I made to the docs, https://github.com/influxdata/telegraf/commit/777b84d1dca12bbe066303ee27d12cf0c9bdf764, I hope this will connect the measurement filtering to the plugins more directly.

Reply:
I like the changes to the docs

Reply:
One thing that would be nice are examples of the use of the metrics filtering options groups under the metrics filtering options section. There are enough examples of their use along CONFIGURATION.md, but since they are spread out consider centralizing them if you are interested on making the docs extra friendly.